tools/gstf_preparation/gstf_preparation.py

```diff
@@ -242,32 +246,51 @@ def write_gene_dict_to_db(conn, gene_dict):
         if gene is None:
             # This can happen when loading a JSON file from Ensembl
             continue
+        if 'confidence' in gene and gene['confidence'] != 'high':
```

Comment:
Is `confidence` an attribute found in Ensembl GFF3 files? Never saw it before (same for `representative` for transcripts).